### DISCUSSION TITLE
🌟 Convert components to a container query approach

### DISCUSSION THREAD

**Discussion by fraincs** [gsoft-inc/sg-orbit#1116] [Ideas]
## Description

As the web is evolving we can now start to react to the width available by a component versus by the width of the viewport, opening doors to documenting components behaviour in reaction to their own dimensions. E.g. how would a Card react to being 200px wide? 1000px wide?

## Checklist

- [ ] Figma
- [ ] Documentation
- [ ] Design respect accessibility specs

**Comment by fraincs** [comment]
https://developer.chrome.com/blog/has-with-cq-m105 Chrome 105 added support for container queries, we're getting there

**Comment by patricklafrance** [comment]
Is this still considered? If so, do you see yourself working on this in the next 6 months?

**Comment by fraincs** [comment]
https://web.dev/cq-stable/ Container queries are now in all 3 major browsers. Going forward is most likely the way to go.
